samples/server/petstore/php-slim/lib/Model/Dog.php

```diff
@@ -10,6 +10,9 @@
 class Dog
 {
     
+    /** @var string $breed */
+    private $breed;
```

Comment:
And here, duplicate of variable `$breed`.

Reply:
@wing328 yup, I have a fix to clean up the duplicates. Thanks for spotting the issue.